scripts/test.canisters.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+POCKET_IC_SERVER_VERSION=6.0.0
+POCKET_IC_SERVER_PATH="pocket-ic"
```

Comment:
Given you are dealing with paths here, what about making sure that the working directory really is the repository directory?

Reply:
We also usually set flags to fail early on error:
```
set -euo pipefail
```

Reply:
Good idea, done!